modules/simul/src/main/JsonView.scala

```diff
@@ -108,6 +110,7 @@ final class JsonView(
         .add("title" -> light.map(_.title))
         .add("provisional" -> ~player.provisional)
         .add("patron" -> light.??(_.isPatron))
+        .add("online" -> true)
```

Comment:
this looks weird, pretending that a player is online when in fact... we don't know?

Reply:
Kept it like that because, of how it was in the original typescript code

https://github.com/ornicar/lila/blob/8cadbf19d455baab93fddd49bb75885508e20f7b/ui/simul/src/view/util.ts#L5-L22

The code that would take into the user's online status would look like this

```scala
  private def playerJson(player: SimulPlayer): Fu[JsObject] =
    getLightUser(player.user) map { light =>
      Json
        .obj(
          "id"     -> player.user,
          "rating" -> player.rating
        )
        .add("name" -> light.map(_.name))
        .add("title" -> light.map(_.title))
        .add("provisional" -> ~player.provisional)
        .add("patron" -> light.??(_.isPatron))
        .add("online" -> isOnline(player.user))
    }
```
With the following patch
```diff
diff --git a/modules/simul/src/main/JsonView.scala b/modules/simul/src/main/JsonView.scala
index e83b878f83..26895acc80 100644
--- a/modules/simul/src/main/JsonView.scala
+++ b/modules/simul/src/main/JsonView.scala
@@ -110,7 +110,7 @@ final class JsonView(
         .add("title" -> light.map(_.title))
         .add("provisional" -> ~player.provisional)
         .add("patron" -> light.??(_.isPatron))
-        .add("online" -> true)
+        .add("online" -> isOnline(player.user))
     }
 
   private def applicantJson(app: SimulApplicant): Fu[JsObject] =
   ```